### PR TITLE
TM-3846 disable pma status dd

### PR DESCRIPTION
**Pull request by andrews-john**
disabled pma status dd using variable. couldnt just use true bc picky doesnt like that

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
> disabled pma status dd using variable. couldnt just use true bc picky doesnt like that

try just no Boolean value

File: src/Components/Panel/PanelMeetingSearch/PanelMeetingSearch.jsx [hunk 1]
```diff
@@ -280,7 +281,7 @@ const PanelMeetingSearch = ({ isCDO }) => {
                   onChange={setSelectedMeetingStatus}
                   valueKey="code"
                   labelKey="description"
-                  disabled={isLoading}
+                  disabled={disabled_dd}
```
Comment (elizabeth-jimenez) [inline src/Components/Panel/PanelMeetingSearch/PanelMeetingSearch.jsx:284]: just `disabled`

Comment (andrews-john) [reply src/Components/Panel/PanelMeetingSearch/PanelMeetingSearch.jsx:284]: oooh very nifty

**Comment by SidelineCory24** [comment]
@andrews-john per the JIRA ticket this disable functionality should be on the Panel Meeting Agendas page, not Panel Meeting Search.

image

**Review by pveland621 — APPROVED** [review]
Verified correct dropdown editing is disabled on my local

**Pull request merged**